### ISSUE TITLE
[Action] Identify the users for the stories and data

### ISSUE THREAD

**Issue by vallss** [cncf/tag-env-sustainability#324]
#### Description
Following the [comment](https://github.com/cncf/tag-env-sustainability/discussions/277#discussioncomment-7635965) we should identify the users for the user stories as a starting point. Some examples could be Open Telemetry, KEDA, GSF. 

### Outcome
The first thing to do would be choosing the users in order to gather information that might be shared in social media, blog or even TAG meetings.

**Comment by nikimanoledaki** [comment]
I'm thrilled to see that you'd like to take this on! I'm here to help :)

We can start by spreading the word with a call to action to source stories from end users.

A few ideas for comms:
- Add something to the next agenda of the regular TAG meeting to ask if anyone has wins/losses/user stories that they would like to share
- Post in the #tag-environmental-sustainability channel asking there as well
- Post on LinkedIn + Twitter/X (@guidemetothemoon could help with this)
- Reach out to @musingvirtual about how to contact the End User group and how they collect user stories in the OpenTelemetry group to learn from their tactics
  - I believe @leonardpahlke said he could help with the End Group as well

**Comment by vallss** [comment]
About the format that could be used for user story + data I ended up with some sort of a template like the following:
```
- Short description of the project we collect information and data from
- Priorities of the project (i.e. metrics that are used and taken into consideration) and motivations
- Results, what were the achievements (including numbers and before/after short analysis)
- How the steps above would help the cloud ecosystem (or more in general why should people/companies take the steps above and choose to include this kind of work) 
```
Using a template we could share it with folks that would like to be involved (GSF, OpenTelemetry, etc.) in async way and then with those information maybe write blogposts or even documentation. However, I really like the idea of planning demos and more "live" events.

**Comment by guidemetothemoon** [comment]
I can definitely help out on getting call to action published on the TAG's social media next week - I will link to this issue and the contact info of the TAG so that relevant people have a way to coordinate with us. Please let me know if anything specific should be included in the publications @vallss. We can also publish multiple times about this.

@vallss @nikimanoledaki would you both like to be added as assignees to coordinate the progress on this issue or just one of you or someone else?😊

**Comment by guidemetothemoon** [comment]
Of course, @vallss ! I will assign this to you for now and will gladly support you on the way. In the meantime we'll see if we can get other contributors to join on this as well😊

**Comment by werstatyw** [comment]
will share my first steps soon, there's something to use less-powerfull vms for k8s provisioning - this is a starting point.
